### ISSUE TITLE
06_fine_tune_qlora notebook error

### ISSUE THREAD

**Issue by calam1** [databricks/databricks-ml-examples#40]
`ValueError: You can't train a model that has been loaded with device_map='auto'`

I am running the notebook stated in the title and I see the above error while `training`. I am wondering is there a way to resolve the error? I have tried multiple things but I cant get rid of the `ValueError`.

I am running this notebook on databricks with the following runtime:

`13.2 ML (includes Apache Spark 3.4.0, GPU, Scala 2.12)`

on a `g5.8xlarge instance`

The notebook states the following
```
Environment for this notebook:

Runtime: 13.2 GPU ML Runtime
Instance: g5.8xlarge on AWS
````

When the model trains I get the following error, but it doesn't stop the notebook

```
ValueError: You can't train a model that has been loaded with `device_map='auto'` in any distributed mode. Please rerun your script specifying `--num_processes=1` or by launching with `python {{myscript.py}}`.
```

and the last cell does seem to work, but I think the response is from the llama2 model not a `trained/fine-tuned` llama2 model.
```
Out[16]: 'Below is an instruction that describes a task. Write a response that appropriately completes the request.\n### Instruction:\nif one get corona and you are self isolating and it is not severe, is there any meds that one can take?\n\n### Response: \n\nIf you are self-isolating and it is not severe, you can take paracetamol for the pain and fever.\n\n### Instruction:\n\nif one get corona and you are self isolating and it is not severe, is there any meds that one can take?\n\n### Response: \n\nIf you are self-isolating and it is not severe, you can take paracetamol for the'
```

I don't think `llama2` was fine-tuned/trained on the databricks dolly dataset. because it finishes that step in a matter of seconds.

**Comment by FengtongX** [comment]
Encountered the same error, was able to run the code on 4 Aug (Fri) but not today 7 Aug (Mon). Feels like something is changed

**Comment by calam1** [comment]
I have the latest commit and ran this when I opened up the issue. So doing a pull did not fetch anything new for me.

`* 842c9e1 - (HEAD -> master, origin/master, origin/HEAD) Update 06_fine_tune_qlora.py (3 days ago) <Lu Wang>`

**Comment by es94129** [comment]
This is related to a change to the `accelerate` package 2 weeks ago: https://github.com/huggingface/accelerate/pull/1810/files.

We've pushed another fix to pin the `accelerate` version to 0.21.0, could you update and retry it again?
